install_ovsdb_etcd.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o xtrace
+set -o errexit
+
+if [[ "${USE_OVSDB_ETCD}" = "yes" ]]; then
+    mkdir -p /root/
+
+    #install Go v1.16
```

Comment:
Nit: missing space after `#` and I guess more accurate would be "Install latest Go version.", right?